src/cmd/ksh93/data/builtins.c

```diff
@@ -41,18 +41,13 @@
 
 #ifndef SHOPT_CMDLIB_DIR
 #define SHOPT_CMDLIB_DIR SH_CMDLIB_DIR
-#endif  // SHOPT_CMDLIB_DIR
+#else
 #ifndef SHOPT_CMDLIB_HDR
 #define SHOPT_CMDLIB_HDR <cmdlist.h>
-#endif  // SHOPT_CMDLIB_HDR
+#endif // SHOPT_CMDLIB_HDR
+#endif // SHOPT_CMDLIB_DIR
 
-#if _AST_VERSION < 20121001L
```

Comment:
This seems dubious to me. There are other places in the code that compare `_AST_VERSION` against various magic values. Shouldn't removing this test be done separate from enabling these builtins?

Reply:
I actually left a comment in the description of https://github.com/att/ast/pull/474/commits/f047655c02d943551a76911c4181f3b02f3a0652 but it may not be clear enough. So I will leave a comment in the source code.